### PR TITLE
General fixups and improvements

### PR DESCRIPTION
**Pull request by TheBearodactyl**
1. Updated crate dependencies
2. Cleaned up project root by moving the default config files to their own dedicated directory
3. Updated `README.md` and `src/lib.rs` to reflect the changes made by change number 2

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [package]
 name = "cuteness"
-version = "0.9.9"
+version = "1.0.0"
```
Comment (blyxyas) [inline Cargo.toml]: We're not 1.0.0 yet, we cannot guarantee that stability

```suggestion
version = "0.9.10"
```

**Review by blyxyas — COMMENTED** [review]
Meow meow

File: Cargo.toml (hunk 1 above)
Comment (TheBearodactyl) [reply Cargo.toml]: sorry about that, incrementing versions is a bad habit of mine

**Review by blyxyas — APPROVED** [review]
LGTM, thanks! :heart: I'll publish it

File: Cargo.toml [hunk 2]
```diff
@@ -13,9 +13,7 @@ include = [
 	"Cargo.lock",
 	"README.md",
 	"src/*",
-	"cuteconfig.default.toml",
-	"SUMMARY.default.toml",
-	"introduction.default.md",
+  "defaults/*",
 	"assets/*"
 ]
```
Comment (blyxyas) [inline Cargo.toml]: Could you add the `.idea` directory to the `exclude` field?

Comment (TheBearodactyl) [reply Cargo.toml]: ye

**Comment by TheBearodactyl** [comment]
oki, it's fixed.

**Comment by TheBearodactyl** [comment]
the fuck.

**Pull request closed without merging**